store/tikv/gcworker/gc_worker.go

```diff
@@ -222,7 +224,7 @@ func (w *GCWorker) leaderTick(ctx goctx.Context) error {
 	}
 
 	w.gcIsRunning = true
-	log.Infof("[gc worker] %s starts GC job, safePoint: %v", w.uuid, safePoint)
+	log.Infof("[gc worker] %s starts the whole job, safePoint: %v", w.uuid, safePoint)
```

Comment:
??

Reply:
I change the log because it is too similar to the log in the func `doGC`.